packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -186,8 +186,10 @@ class MenuToggleBase extends React.Component<MenuToggleProps> {
             aria-label={ariaLabel}
             disabled={isDisabled}
             onClick={onClick}
+            {...(children && { style: { display: 'flex', paddingLeft: '8px' } })}
```

Comment:
One little nit - this is the "8px" spacer, except it's a `rem` unit (0.5rem).

```suggestion
            {...(children && { style: { display: 'flex', paddingLeft: 'var(--pf-v5-global--spacer--sm)' } })}
```